fetcher/jvn/xml/jvn.go

```diff
@@ -4,15 +4,20 @@ import (
 	"encoding/json"
 	"encoding/xml"
 	"fmt"
+	"net/http"
+	"net/url"
+	"runtime"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/PuerkitoBio/goquery"
```

Comment:
__[go vet]__ cannot find package "github.com/PuerkitoBio/goquery" in any of: ([link](https://sider.review/gh/repos/54628930/pulls/129#issue-4523446))

<sub>posted by Sider</sub>